cli/docker_build.py

```diff
@@ -184,7 +184,7 @@ def null_swaparoo():
 
 class Docker(object):
 
-    def docker_build(self, dockerUtilsObj, appObj, directory, repo, projects, path, image_tag, docker_file='Dockerfile'):
+    def docker_build(self, dockerUtilsObj, appObj, directory, repo, projects, path, image_tag, build_args, docker_file='Dockerfile'):
```

Comment:
But you control and converted all the calls to docker_build so you dont need to set a default, right?